src/hll.c

```diff
@@ -77,11 +93,21 @@ static uint8_t g_output_version = 1;
 #define DEFAULT_EXPTHRESH	-1
 #define DEFAULT_SPARSEON	1
 
+#define HLL_EXTENSION_NAME "hll"
+#define HLL_ADD_AGG_NAME "hll_add_agg"
+#define HLL_UNION_AGG_NAME "hll_union_agg"
+
 static int32 g_default_log2m = DEFAULT_LOG2M;
 static int32 g_default_regwidth = DEFAULT_REGWIDTH;
 static int64 g_default_expthresh = DEFAULT_EXPTHRESH;
 static int32 g_default_sparseon = DEFAULT_SPARSEON;
 
+static void
+multi_aggregation_restriction_hook(PlannerInfo *root, UpperRelationKind stage,
```

Comment:
We use multi_ prefix in Citus to highlight "distributedness" (which we are not using as much as before even in Citus).

Reply:
perhaps make it `hll_aggregation_restriction_hook`